packages/nc-gui-v2/components/template/Editor.vue

```diff
@@ -261,20 +259,30 @@ async function importTemplate() {
   } catch (e: any) {
     toast.error(await extractSdkResponseErrorMsg(e))
   } finally {
-    // TODO: close dialog when the integration is ready
     isImporting.value = false
   }
 }
+
+const isValid = () => {
+  for (const [_, o] of Object.entries(validateInfos)) {
+    if (o?.validateStatus) {
+      if (o.validateStatus === 'error') {
+        return false
+      }
+    }
+  }
+  return true
+}
```

Comment:
Can this not be a computed property?

Reply:
You mean turning this to be a computed property?